### ISSUE TITLE
New Release?

### ISSUE THREAD

**Issue by brassier** [auth0/auth0-cordova#200]
It looks like there have been commits to patch up vulnerabilities in Feb/March 2022.  I don't see a new release that picks these up though.  Is a new release planned?

**Comment by stevehobbsdev** [comment]
Yes, looks like a new release is overdue - I will put one out this week 👍🏻
